src/client/gui/lib/main.dart

```diff
@@ -21,15 +22,24 @@ import 'vm_table/vm_table_screen.dart';
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
-  await setupLogger();
+  // Get the current screen size
+  final screen = await getCurrentScreen();
```

Comment:
Why is this removing the logger setup?

Also, since we only care about the screen size, not the entire screen object, you could do something like
```dart
final screenSize = await getCurrentScreen().then((screen) {
    return screen?.frame.size;
});
```
so that you don't have to do the `.frame.size` thing multiple times